src/python/CRABInterface/DataWorkflow.py

```diff
@@ -311,6 +312,10 @@ def resubmit2(self, workflow, publication, jobids, siteblacklist, sitewhitelist,
             msg = "You cannot resubmit %s if the task is in status %s." % (resubmitWhat, task_status)
             raise ExecutionError(msg)
 
+        if task_status == 'KILLED' and task_splitting == 'Automatic':
```

Comment:
Do we want `crab kill` to be final **only** for Automatic splitting mode?

Reply:
I tend to forget what our conclusion about that was. It's an easy change, though.